.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
-  - 3.6
+  - "2.7"
```

Comment:
The tests 'just' passed... Looks like we are lucky.
This is a good sign, as i remember that we had some strange problems with unstable list ordering in different python versions...